onnxruntime/contrib_ops/cpu/word_conv_embedding.cc

```diff
@@ -16,13 +16,14 @@ void WordConvEmbedding::CharEmbeddingLookup(
     size_t seq_len,
     size_t word_len,
     size_t char_embedding_size,
+    size_t filter_width,
     const int* words_len_ptr,
     float* dst) const {
   for (size_t word_inx = 0; word_inx < seq_len; word_inx++) {
     if (words_len_ptr[word_inx] > 0) {
       const int* cur_seq_ptr = seq_ptr + word_inx * word_len;
       float* cur_dst_ptr = dst + word_inx * word_len * char_embedding_size;
-      for (size_t char_inx = 0; char_inx < word_len; char_inx++) {
+      for (size_t char_inx = 0; char_inx < std::max<size_t>(words_len_ptr[word_inx], filter_width); char_inx++) {
```

Comment:
I Don't know if the compiler is able to detect if std::max<size_t>(words_len_ptr[word_inx], filter_width) is modified by the loop or not and then avoid computing it at each iteration. If not, maybe it is worth moving it before the loop.